lib/reporting/report_row_builder.rb

```diff
@@ -27,7 +27,7 @@ def build_row(item)
     end
 
     def slice_and_format_row(row)
-      result = row.to_h.reject { |k, _v| k.in?(report.fields_to_hide) }
+      result = row.to_h.select { |k, _v| k.in?(report.fields_to_show) }
```

Comment:
Not sure on how to handle this one with the `group_by` option.
Currently, the spec `spec/lib/reports/report_spec.rb:153` fails, because of grouping option.

```diff
@@ -1,10 +1,10 @@
        [{:header=>"Hub 1"},
         {:header=>"Abby"},
       - {:product=>"Orange", :quantity=>6},
       + {:customer=>"Abby", :hub=>"Hub 1", :product=>"Orange", :quantity=>6},
         {:header=>"John"},
       - {:product=>"Apple", :quantity=>4},
       + {:customer=>"John", :hub=>"Hub 1", :product=>"Apple", :quantity=>4},
         {:header=>"Hub 2"},
         {:header=>"John"},
       - {:product=>"Pear", :quantity=>3},
       - {:product=>"Apple", :quantity=>5}]
       + {:customer=>"John", :hub=>"Hub 2", :product=>"Pear", :quantity=>3},
       + {:customer=>"John", :hub=>"Hub 2", :product=>"Apple", :quantity=>5}]
```

We should _remove_ or _unselect_ columns that are grouped (such as `hub` and `customer`) in the spec. 

If @seballot you could point me something... Thanks a lot! 🙏